package.json

```diff
@@ -41,6 +41,7 @@
     "preview": "vite preview",
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
+    "lint:staged": "eslint $(git diff --name-only --cached | xargs)",
```

Comment:
Show me the lint in my staged files.

Reply:
It will convert line broken output to the  space broken output that the eslint command expected.